docs/features/EXTRACTION_RULES.md

```diff
@@ -113,3 +118,121 @@ Value can be anything except `null`.
 
 The source that Crawler will try to extract content from.
 Currently only `html` or `url` is supported.
+
+## Examples
+
+### Extracting from HTML
+
+I have a simple website for an RPG.
+A page describing cities in the RPG is hosted at `https://totally-real-rpg.com/cities`.
+The HTML for this page looks like this:
+
+```HTML
+<!DOCTYPE html>
+<html>
+  <body>
+    <div>Cities:</div>
+    <div class="city">Neverwinter</div>
+    <div class="city">Waterdeep</div>
+    <div class="city">Menzoberranzan</div>
```

Comment:
```suggestion
    <div class="city">Summerstay</div>
    <div class="city">Drenchwell</div>
    <div class="city">Mezzoterran</div>
```

Reply:
Just suggesting nonsense names instead of Forgotten Realms cities :)

Feel free to use something else!

Reply:
😢 I thought the pop references were cute

Reply:
I'll save them for unit tests :D